neurolib/models/multimodel/builder/adex.py

```diff
@@ -0,0 +1,944 @@
+
+        exc_inp, inh_inp, exc_inp_sq, inh_inp_sq = self._compute_couplings(coupling_variables)
+
+        I_sigma = se.sqrt(
+            2
+            * self.params["J_exc_max"] ** 2
+            * I_syn_sigma_exc
+            * self.params["tau_syn_exc"]
+            * self.params["tau_m"]
+            / ((1.0 + exc_inp) * self.params["tau_m"] + self.params["tau_syn_exc"])
+            + 2
+            * self.params["J_inh_max"] ** 2
+            * I_syn_sigma_inh
+            * self.params["tau_syn_inh"]
+            * self.params["tau_m"]
+            / ((1.0 + inh_inp) * self.params["tau_m"] + self.params["tau_syn_inh"])
+            + self.params["sigma_ext"] ** 2
+        )
+
+        # get values from linear-nonlinear lookup table
+        firing_rate_now = self.callback_functions["firing_rate_lookup"](I_mu, I_sigma)
+        tau = self.callback_functions["tau_lookup"](I_mu, I_sigma)
+
+        d_I_mu = (
+            self.params["J_exc_max"] * I_syn_mu_exc
+            + self.params["J_inh_max"] * I_syn_mu_inh
+            + system_input(self.noise_input_idx[0])
+            + self.params["ext_current"]
+            - I_mu
+        ) / tau
+
+        d_I_syn_mu_exc = ((1.0 - I_syn_mu_exc) * exc_inp - I_syn_mu_exc) / self.params["tau_syn_exc"]
+
+        d_I_syn_mu_inh = ((1.0 - I_syn_mu_inh) * inh_inp - I_syn_mu_inh) / self.params["tau_syn_inh"]
+
+        d_I_syn_sigma_exc = (
+            (1.0 - I_syn_mu_exc) ** 2 * exc_inp_sq
+            + (exc_inp_sq - 2.0 * self.params["tau_syn_exc"] * (exc_inp + 1.0)) * I_syn_sigma_exc
+        ) / (self.params["tau_syn_exc"] ** 2)
+
+        d_I_syn_sigma_inh = (
+            (1.0 - I_syn_mu_inh) ** 2 * inh_inp_sq
+            + (inh_inp_sq - 2.0 * self.params["tau_syn_inh"] * (inh_inp + 1.0)) * I_syn_sigma_inh
+        ) / (self.params["tau_syn_inh"] ** 2)
+        # firing rate as dummy dynamical variable with infinitely fast
+        # fixed-point dynamics
+        d_firing_rate = -self.params["lambda"] * (firing_rate - firing_rate_now)
+
+        return [
+            d_I_mu,
+            d_I_syn_mu_exc,
+            d_I_syn_mu_inh,
+            d_I_syn_sigma_exc,
+            d_I_syn_sigma_inh,
+            d_firing_rate,
+        ]
+
+
+class AdExNode(SingleCouplingExcitatoryInhibitoryNode):
+    """
+    Default AdEx network node with 1 excitatory (featuring adaptive current) and
+    1 inhibitory population.
+    """
+
+    name = "AdEx mean-field node"
+    label = "AdExNode"
+
+    sync_variables = [
+        "node_exc_exc",
+        "node_inh_exc",
+        "node_exc_inh",
+        "node_inh_inh",
+        # squared variants
+        "node_exc_exc_sq",
+        "node_inh_exc_sq",
+        "node_exc_inh_sq",
+        "node_inh_inh_sq",
+    ]
+
+    default_network_coupling = {
+        "network_exc_exc": 0.0,
+        "network_exc_exc_sq": 0.0,
+    }
+
+    default_output = f"q_mean_{EXC}"
+
+    def _rescale_connectivity(self):
```

Comment:
and again - the local E-I connectivity in ALN model is rescaled... so rescaling here